code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -899,26 +899,55 @@
 
 /datum/reagent/medicine/earthsblood //Created by ambrosia gaia plants
 	name = "Earthsblood"
-	description = "Ichor from an extremely powerful plant. Great for restoring wounds, but it's a little heavy on the brain."
+	description = "Ichor from an extremely powerful plant. Great for restoring wounds, but it's a little heavy on the brain. For some strange reason, it also induces temporary pacifism in those who imbibe it and semi-permanent pacifism in those who overdose on it."
 	color = rgb(255, 175, 0)
+	metabolization_rate = REAGENTS_METABOLISM //Even though metabolization_rate is already REAGENTS_METABOLISM by default, I was asked told to redefine it here, so I did. It's just one redundant line; it can't hurt much, can it?
 	overdose_threshold = 25
 
 /datum/reagent/medicine/earthsblood/on_mob_life(mob/living/carbon/M)
-	M.adjustBruteLoss(-3 * REM, 0)
-	M.adjustFireLoss(-3 * REM, 0)
-	M.adjustOxyLoss(-15 * REM, 0)
-	M.adjustToxLoss(-3 * REM, 0)
-	M.adjustOrganLoss(ORGAN_SLOT_BRAIN, 2 * REM, 150) //This does, after all, come from ambrosia, and the most powerful ambrosia in existence, at that!
-	M.adjustCloneLoss(-1 * REM, 0)
-	M.adjustStaminaLoss(-30 * REM, 0)
-	M.jitteriness = min(max(0, M.jitteriness + 3), 30)
+	if(current_cycle <= 25) //10u has to be processed before u get into THE FUN ZONE
+		M.adjustBruteLoss(-1 * REM, 0)
+		M.adjustFireLoss(-1 * REM, 0)
+		M.adjustOxyLoss(-0.5 * REM, 0)
+		M.adjustToxLoss(-0.5 * REM, 0)
+		M.adjustCloneLoss(-0.1 * REM, 0)
+		M.adjustStaminaLoss(-0.5 * REM, 0)
+		M.adjustOrganLoss(ORGAN_SLOT_BRAIN, 1 * REM, 150) //This does, after all, come from ambrosia, and the most powerful ambrosia in existence, at that!
+	else
+		M.adjustBruteLoss(-5 * REM, 0) //slow to start, but very quick healing once it gets going
+		M.adjustFireLoss(-5 * REM, 0)
+		M.adjustOxyLoss(-3 * REM, 0)
+		M.adjustToxLoss(-3 * REM, 0)
+		M.adjustCloneLoss(-1 * REM, 0)
+		M.adjustStaminaLoss(-3 * REM, 0)
+		M.jitteriness = min(max(0, M.jitteriness + 3), 30)
+		M.adjustOrganLoss(ORGAN_SLOT_BRAIN, 2 * REM, 150)
+		if(prob(10))
+			M.say(pick("Yeah, well, you know, that's just, like, uh, your opinion, man.", "Am I glad he's frozen in there and that we're out here, and that he's the sheriff and that we're frozen out here, and that we're in there, and I just remembered, we're out here. What I wanna know is: Where's the caveman?", "It ain't me, it ain't me...", "Make love, not war!", "Stop, hey, what's that sound? Everybody look what's going down...", "Do you believe in magic in a young girl's heart?"), forced = /datum/reagent/medicine/earthsblood)
```

Comment:
Should I add a comma after "magic"? The lyrics I've seen for that song don't have a comma there, but the way the song itself is performed/written makes it sound like there is one there.

Reply:
Also, should I replace "Stop, hey, what's that sound? Everybody look what's going down..." with "There's something happening here..."? The former is a more memorable line from the song, but the latter is more likely to sound like an appropriate/relevant thing to say in a situation (in other words, it's more likely to be misinterpreted by others as having been said voluntarily by the person high on earthsblood (instead of as a thing that earthsblood forced them to say)).

Reply:
This could be a json file in [strings].

[strings]: https://github.com/tgstation/tgstation/tree/72edf6cea17c167cfdb02d822f4ed3188ec4f1c1/strings

Reply:
I just copied the way spray tan did it.